README.rst

```diff
@@ -182,480 +182,594 @@ automatically fix anything:
 
 Choose from the list of available rules:
 
-* **array_syntax**
-   | PHP arrays should be declared using the configured syntax (requires PHP
-   | >= 5.4 for short syntax).
-   | *Rule is: configurable.*
+**array_syntax**
```

Comment:
Because I was not sure that having the configuration options as a sublist would be nice, but I checked and it's ok, so I restored the bullets.